### PR TITLE
CEM-1063 Update List and Navigation

### PR DESCRIPTION
**Pull request by Ampatte2**
ListHeader and ListFooter padding as optional props.

Changed isSelected background color to same shade as onHover.

Setting display of ListToggle as a styled higher order component does not work as intended. Added logic to hide the toggle on responsive.

Added zindex and padding props to Navigation

### CODE REVIEW

File: src/Containers/List/List.tsx [hunk 1]
```diff
@@ -16,7 +16,6 @@ export interface ListProps extends React.HTMLAttributes<HTMLDivElement> {
     left?: string;
     onCloseTranslateXAxis?: string;
     cssPosition?: string;
-    isToggleable?: boolean;
```
Comment (ralph-dev) [inline src/Containers/List/List.tsx:19]: Why remove?

File: src/Containers/List/ListToggle.tsx [hunk 2]
```diff
@@ -10,12 +10,14 @@ const LIST_TOGGLE_LEFT = AngleLeft;
 interface ListToggleProps extends ButtonProps {
     isOpen: boolean;
     setIsOpen: React.Dispatch<React.SetStateAction<boolean>>;
+    hideToggleTablet?: boolean;
```
Comment (ralph-dev) [inline src/Containers/List/ListToggle.tsx:13]: IsToggleTabletHidden

Always start with is for Boolean.

Comment (ralph-dev) [reply src/Containers/List/ListToggle.tsx:13]: IsHiddenTabletToggle etc

File: src/Containers/List/List.tsx (hunk 1 above)
Comment (ralph-dev) [reply src/Containers/List/List.tsx:19]: Maybe isFrozen or isHideable

**Review by ralph-dev — APPROVED** [review]
Lgtm

File: src/Containers/List/List.tsx (hunk 1 above)
Comment (Ampatte2) [reply src/Containers/List/List.tsx:19]: It serves no purpose, functionality is effectively replaced by toggleComponent. Pass setIsOpen to either togglecomponent or to listItem to control toggled state.

File: src/Containers/List/ListToggle.tsx (hunk 2 above)
Comment (Ampatte2) [reply src/Containers/List/ListToggle.tsx]: Changed to isToggleHiddenDesktop

**Comment by ralph-dev** [comment]
all good to merge

**Pull request merged**